src/lib/components/SelectField/_components/OptionField.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const OptionField = ({
+  id,
+  option,
+}) => (
+  <option
+    disabled={option.disabled}
+    id={id && `${id}__item__${option.value}`}
+    value={option.value}
+  >
+    {option.label}
+  </option>
+);
+
+OptionField.defaultProps = {
+  id: undefined,
+};
+
+OptionField.propTypes = {
+  /**
+   * ID of the input HTML element.
+   *
+   * Serves as a prefix for individual options:
+   * * `<ID>__item__<VALUE>`
+   */
+  id: PropTypes.string,
+  /**
+   * Set of options to be chosen from.
+   *
+   * Both option or optgroup are accepted.
+   */
+  option: PropTypes.shape({
```

Comment:
There is inconsistence between code and documentation. I think `Both option or optgroup are accepted.` should removed from here as it is not true.